Clean/GadgetsNew/Add8/Addition8Full.lean

```diff
@@ -14,10 +15,7 @@ namespace Add8Full
 variable {p : ℕ} [Fact (p ≠ 0)] [Fact p.Prime]
 variable [p_large_enough: Fact (p > 512)]
 
-open Circuit
 open Provable (field field2 fields)
-open ByteLookup
-open Expression
```

Comment:
I didn't like that we had to open so much stuff, I feel like the core operations of a framework should be available just by importing the framework